### PR TITLE
Ability to enable/disable logging

### PR DESCRIPTION
**Pull request by weikinhuang**

### CODE REVIEW

**Comment by weikinhuang** [comment]
@DFoxinator please take a look at this

File: src/devtools.html [hunk 1]
```diff
@@ -5,10 +5,11 @@
 </head>
 <body>
 
-<script src="vendor/q/q.js"></script>
+<!-- <script src="vendor/q/q.js"></script> -->
```
Comment (DFoxinator) [inline src/devtools.html:8]: Did you mean to do this or should it be deleted?

Comment (weikinhuang) [inline src/devtools.html:8]: yes, will be put back in later when i mess with post message

**Comment by DFoxinator** [comment]
+1

**Pull request merged**